src/main/java/org/example/models/Diagnosis.java

```diff
@@ -0,0 +1,85 @@
+package org.example.models;
+import jakarta.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "diagnosis")
+public class Diagnosis {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @ManyToOne
+    @JoinColumn(name = "doc_id", nullable = false)
+    private Doctor doctor;
+
+    @Column(name = "symptoms", nullable = false)
+    private String symptoms;
+
+    @ManyToOne
+    @JoinColumn(name = "patient_id", nullable = false)
+    private Patient patient;
+
+    @Column(name = "date", nullable = false)
+    private LocalDate date;
```

Comment:
encounterDate/visitDate to make it more clear

Reply:
Renamed according to the next comment